router/pkg/config/fixtures/full.yaml

```diff
@@ -212,6 +212,7 @@ engine:
   epoll_kqueue_conn_buffer_size: 128
   websocket_read_timeout: "5s"
   execution_plan_cache_size: 1024
+  resolver_max_recyclable_parser_size: 32768
```

Comment:
Where is this number coming from?

Reply:
This is the number of cached json values being parsed by the Parser. The Parser itself doubles the size of the underlying array every time it reaches the limit so a power of 2 is usually a good threshold imho.